src/main/kotlin/no/nav/familie/ba/sak/kjerne/grunnlag/personopplysninger/Person.kt

```diff
@@ -139,6 +139,37 @@ data class Person(
         val nåværendeMedlemskap = finnNåværendeMedlemskap(statsborgerskap)
         return finnSterkesteMedlemskap(nåværendeMedlemskap)
     }
+
+    fun tilKopiForNyttPersonopplysningGrunnlag(nyttPersonopplysningGrunnlag: PersonopplysningGrunnlag): Person =
+        copy(
+            id = 0,
+            personopplysningGrunnlag = nyttPersonopplysningGrunnlag,
+            bostedsadresser = mutableListOf(),
+            statsborgerskap = mutableListOf(),
+            opphold = mutableListOf(),
+            arbeidsforhold = mutableListOf(),
+            sivilstander = mutableListOf(),
+        )
+            .also {
+                it.bostedsadresser.addAll(
+                    bostedsadresser.map { grBostedsadresse ->
+                        grBostedsadresse.tilKopiForNyPerson(
+                            it,
+                        )
+                    },
+                )
+                it.statsborgerskap.addAll(
+                    statsborgerskap.map { grStatsborgerskap ->
+                        grStatsborgerskap.tilKopiForNyPerson(
+                            it,
+                        )
+                    },
+                )
+                it.opphold.addAll(opphold.map { grOpphold -> grOpphold.tilKopiForNyPerson(it) })
+                it.arbeidsforhold.addAll(arbeidsforhold.map { grArbeidsforhold -> grArbeidsforhold.tilKopiForNyPerson(it) })
+                it.sivilstander.addAll(sivilstander.map { grSivilstand -> grSivilstand.tilKopiForNyPerson(it) })
+                it.dødsfall = dødsfall?.tilKopiForNyPerson(it)
+            }
```

Comment:
Nesten så disse ulike metodene skulle vært plassert øverst i klassen for synlighet, eventuellt en kommentar på klassene om at man må vurdere nye felt inn i denne kopi-metoden 😬 

Her går det også å slette it og erstatte med person
```suggestion
        ).also { person ->
            bostedsadresser.addAll(bostedsadresser.map { it.tilKopiForNyPerson(person) })
            statsborgerskap.addAll(statsborgerskap.map { it.tilKopiForNyPerson(person) })
            opphold.addAll(opphold.map { it.tilKopiForNyPerson(person) })
            arbeidsforhold.addAll(arbeidsforhold.map { it.tilKopiForNyPerson(person) })
            sivilstander.addAll(sivilstander.map { it.tilKopiForNyPerson(person) })
            dødsfall = dødsfall?.tilKopiForNyPerson(person)
        }
```

Reply:
@blommish Er vel fortsatt nødt til å kjøre `person.bostedsadresser.addAll(...)`?. Jeg er jo interessert i å oppdatere listene i nytt objekt og ikke det gamle?